pom.xml

```diff
@@ -63,6 +63,8 @@
         <artifactId>gatling-maven-plugin</artifactId>
         <version>${gatling-maven-plugin.version}</version>
         <configuration>
+          <!-- Default folder when running the recorder with the 'gatling:recorder' goal: -->
+          <simulationsFolder>${project.basedir}/src/test/scala</simulationsFolder>
```

Comment:
That's currently the default value in the plugin, but we should probably change that default to Java anyway, so I think it's worth setting it explicitly here.

Reply:
We should be using the `testSourceDirectory` as default value.

Reply:
We should be using the testSourceDirectory as default value.